src/ReactLeafletMap/layers/VectorLayer/Feature/index.js

```diff
@@ -38,7 +52,7 @@ class Feature extends React.PureComponent {
         this.fid = props.fid;
 
         if (props.type === "Point" && props.pointAsMarker) {
-            this.iconId = utils.uuid();
+            this.iconId = this.props.fid ? this.props.fid + "_icon" : utils.uuid();
```

Comment:
I recommend to use string templates `${this.props.fid}_icon` . It is safer.

Reply:
Fixed in next commit